### ISSUE TITLE
JSON output for programmatic usage

### ISSUE THREAD

**Issue by lucafaggianelli** [trap-bytes/hauditor#2]
Hey nice tool! I’m actually looking into a tool like this to integrate it in my devops workflow and periodically check multiple websites, so it would be important to have a json output. What do you think about integrating this feature?

**Comment by trap-bytes** [comment]
Yes I think it could be a valuable adding. I will add it as soon as I find a little bit of time. Of course if you want you can also submit a PR.

**Comment by lucafaggianelli** [comment]
Amazing! I'll try to come up with a PR
